src/components/event-form/event-form-schema.ts

```diff
@@ -10,6 +10,7 @@ const eventFormBaseSchema = z.object({
 const eventFormPartnersSchema = z.object({
 	withPartners: z.literal(true),
 	partners: z.array(z.file()).min(1),
+	partnerType: z.enum(['event-partner', 'sponsor', 'only-logo']),
```

Comment:
Może będzie lepiej, gdy stworzymy enuma `PartnerType` w pliku `event-form-types.ts`? Dzięki temu deklarujemy typ, którego używamy wszędzie i nie ma duplikacji, a dodatkowo będzie można odwołać się do tych wartości w runtime:
```tsx
<Radio value={PartnerType.EventPartner}>Event partner</Radio>
<Radio value={PartnerType.Sponsor}>Sponsor</Radio>
<Radio value={PartnerType.None}>Only logo</Radio>
```
```suggestion
	partnerType: z.nativeEnum(PartnerType),
```

Reply:
z.nativeEnum is deprecated 😅 [but z.enum is supports](https://zod.dev/v4/changelog?id=znativeenum-deprecated)

```ts
export enum PartnerType {
	EventPartner = 'event-partner',
	Sponsor = 'sponsor',
	OnlyLogo = 'only-logo',
}
```

`TS1294: This syntax is not allowed when erasableSyntaxOnly is enabled`